ipatests/pytest_ipa/integration/create_bridge.py

```diff
@@ -0,0 +1,173 @@
+import re
+import textwrap
+
+from ipatests.pytest_ipa.integration import tasks
+
+
+def setup_scim_server(host, version="main"):
+    dir = "/opt/ipa-tuura"
+    password = host.config.admin_password
+    tasks.install_packages(host, ["unzip", "java-11-openjdk-headless",
+                                  "openssl", "maven", "wget", "git",
+                                  "firefox", "xorg-x11-server-Xvfb",
+                                  "python3-pip"])
+
+    # Download ipa-tuura project
+    url = "https://github.com/freeipa/ipa-tuura"
+    host.run_command(["git", "clone", "-b", f"{version}", f"{url}", f"{dir}"])
+
+    # Prepare SSSD config
+    host.run_command(["python3", "./prepare_sssd.py"],
+                     cwd=f"{dir}/src/install")
+
+    # Install django requirements
+    django_reqs = f"{dir}/src/install/requirements.txt"
+    host.run_command(["pip", "install", "-r", f"{django_reqs}"])
+
+    # Prepare models and database
+    host.run_command(["python", "manage.py", "makemigrations", "ipatuura"],
+                     cwd=f"{dir}/src/ipa-tuura")
+    host.run_command(["python", "manage.py", "migrate"],
+                     cwd=f"{dir}/src/ipa-tuura")
+
+    # Add necessary admin vars to bashrc
+    env_vars = textwrap.dedent(f"""
+    export DJANGO_SUPERUSER_PASSWORD={password}
+    export DJANGO_SUPERUSER_USERNAME=scim
+    export DJANGO_SUPERUSER_EMAIL=scim@ipa.test
```

Comment:
Should ipa.test be hardcoded? There are a few other tests with it set as well but I believe this is configurable. It may be better to use host.domain.name.

Reply:
No it should not.  That was an oversight on my part.  I'll fix that.

Reply:
This was fixed.